includes/Controllers/Webhooks/PaymentController.php

```diff
@@ -33,14 +33,21 @@ public function capturePayment() {
 			] ) ) {
 				$error = __( 'The order has been authorized and is awaiting approval.', 'woocommerce' );
 			}
+		}
 
-			if ( ! empty( $order ) ) {
-				$order->calculate_totals();
-			}
+		if ( is_object( $order ) ) {
+			$order->calculate_totals();
+			$orderTotal = $order->get_total();
+		} else {
+			$orderTotal = false;
+		}
 
+		if ( ! empty( $orderTotal ) ) {
+			$amount = $orderTotal;
+		} else {
+			$amount = wc_format_decimal( $_POST['amount'] );
 		}
```

Comment:
What is the purpose of the above fix - what does it do?

Reply:
The amount we are sending to the API should come from the WooCommerce orders data storage, not directly from the HTML 'total price' form input.

I've kept the global $_POST amount as a precaution, in case it is called from another scope (beyond the regular checkout flow).

Reply:
Yep makes sense, thanks.